plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointLabel.tsx

```diff
@@ -91,6 +98,22 @@ const PointLabel: React.FC<BaseFieldProps<"pointLabel">> = ({ value, editMode, o
     [updatePointLabelByProp],
   );
 
+  useEffect(() => {
+    if (!isActive || !dataID) return;
+    const timer = setTimeout(() => {
+      postMsg({
+        action: "updateDatasetInScene",
+        payload: {
+          dataID,
+          update: { marker: { style: "image", ...pointLabel } },
```

Comment:
Same here. Please reference [this](https://github.com/reearth/reearth-web/blob/main/src/core/mantle/types/appearance.ts) for how to write the updates